### DISCUSSION TITLE
New Ride Type: Modern Corkscrew Roller Coaster

### DISCUSSION THREAD

**Discussion by karst** [OpenRCT2/OpenRCT2#21013] [Feature requests]
Based on Vekoma's new generation track type. Think of Lech Coaster at Legendia or Abyssus at Energylandia. 

I think this would be a great addition to the game for modernisation's sake.

Vekoma has explicitly told us not to use their likeness in the game, so vehicles would be up for artistic liberty.

**Comment by spacek531** [comment]
Instead of Vekoma, why not S&S. The track type is similar and I have a vehicle model already made.

**Reply by karst** [reply]
I could definitely see that work too, could have a compressed air launch too

**Reply by Realsteel89** [reply]
building the track would be super easy it's just a narrower version of the lim that we had in the game yet with an skeleton under the rails.
how do you want to achieve a copyright infringement with a pixelated replica without many details - is that not possible or am I wrong as long as you don't use the names?
